content/episodes/2019-03-04-S04E07-what-if-women-built-the-internet.md

```diff
@@ -0,0 +1,234 @@
+---
+date: 2019-03-04T04:55:00-08:00
+title: "What if women built the internet?"
+ogtitle: "What if women built the internet?"
+description: |
+    The more diverse and inclusive we can make the web, the better it is for everyone. With that thought in mind, we asked ourselves "what would the Internet look like if it was built by mostly women?" In this episode, we put that question to a number of women who are involved in tech to get their thoughts. Witchsy founders Kate Dwyer and Penelope Gazin start us off with a story about the stunt they had to pull to get their site launched - and counter the sexist attitudes they fought against along the way. Brenda Darden Wilkerson recalls her life in tech in the 1980s and 90s, and shares her experience leading Anita B, an organization striving to get more women hired in tech.
+
+    Coraline Ada Ehmke created the Contributor Covenant, a voluntary code of conduct being increasingly adopted by the open source community. She explains why she felt it necessary, and how it's been received (spoiler: it brought out the haters); and Mighty Networks CEO Gina Bianchini rolls her eyes at being called a "lady CEO", and tells us why diversifying the board rooms is a no-brainer that is great for business and innovation.
+ogdescription: "Explore why the web needs to welcome everyone so it can be there for every person who needs it."
+number: 30
+season: 4
+seasonepisode: 7
+url: /season4/episode7/
+embed: "27508dd7"
+mp3: "https://dts.podtrac.com/redirect.mp3/audio.simplecast.com/27508dd7.mp3"
+categories: "episodes"
+host: "Manoushz"
+shownotes: |
+  Mozilla is demonstrating its dedication to fostering both an inclusive web and working places. It's making big strides, and invites you to learn more about the journey... [Learn how](https://blog.mozilla.org/inclusion/2018/04/19/diversity-and-inclusion-at-mozilla/)
```

Comment:
The 2018 numbers are slated to be published on March 5, the day after this episode goes out. I asked Tara if we might be able to move that to Monday so we can link to the post here but she doesn't think it'll be ready before Tuesday... maybe we should just link to blog.mozilla.org/inclusion/ here, rather than the post with 2017's data?

Reply:
This is a link to last year's disclosure so I think it's fine.